docs/testingplatform/architecture.md

```diff
@@ -64,19 +69,19 @@ public static class TestApplicationBuilderExtensions
 
 The parameters for the api `AddEnvironmentVariableProvider` will be explained in later sections.
 
-When we run the application this time, the runtime will start the `Contoso.UnitTests.exe` process as shown in the previous sample. However, it will also detect that a test host controller extension is registered. As a result, it will restart another instance of the `Contoso.UnitTests.exe` process as a child process. This is done to properly set the environment variables as required by the extension registered with the `AddEnvironmentVariableProvider` API.
+When we run `Contoso.UnitTests.exe` this time, the testing platform detects that a `TestHostController` extension is registered. As a result, it starts another instance of the `Contoso.UnitTests.exe` process as a child process. This is done to properly set the environment variables as required by the extension registered with the `AddEnvironmentVariableProvider` API.
 
-The in memory processes layout will be:
+The process layout looks like this:
 
 ```mermaid
 graph TD;
     TestHostController:'Contoso.UnitTests.exe'-->TestHost:'Contoso.UnitTests.exe;
 ```
 
 > [!NOTE]
-> The provided example assumes a console application layout, which handles the restart process correctly and propagates all command line arguments to the child process.
+> The provided example assumes a console application layout, which handles the start process correctly and propagates all command line arguments to the child process.
```

Comment:
start, because restart was suggesting that the currently running process will exit and start again, which is not the case.